### PR TITLE
Migrate Error Boundary styles to emotion

### PR DESCRIPTION
**Pull request by Tiago-Zarzavidjian**
#### :page_facing_up: Description:

This PR migrates the styles of the `ErrorBoundary` component to emotion.

---

#### :heavy_check_mark: Tasks:

* Migrate styles.

---

#### :play_or_pause_button: Demo:
image

@loopstudio/react-devs

### CODE REVIEW

File: src/components/ErrorBoundary/ErrorBoundary.js [hunk 1]
```diff
@@ -2,7 +2,13 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { FormattedMessage } from 'react-intl';
 
-import styles from './ErrorBoundary.module.scss';
+import {
+  BtnReportIssue,
```
Comment (mconiglio) [inline src/components/ErrorBoundary/ErrorBoundary.js:6]: maybe we can avoid abbreviations like this one? maybe just `ReportButton` would be enough considering the scope where this button is defined

File: src/components/ErrorBoundary/ErrorBoundary.styles.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+export const ErrorMessage = styled.p`
+  margin-top: 1.25rem;
+  margin-bottom: 1.25rem;
+  width: 50%;
+  text-align: left;
+`;
+
+export const BtnReportIssue = styled.a`
+  background-color: ${({ theme }) => theme.color.indigo600};
+  border-radius: 9999px;
```
Comment (mconiglio) [inline src/components/ErrorBoundary/ErrorBoundary.styles.js:35]: maybe we can use the new style of a not fully rounded button that is used on other screens

File: src/components/ErrorBoundary/ErrorBoundary.styles.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+export const ReportButton = styled.a`
+  background-color: ${({ theme }) => theme.color.indigo600};
+  border-radius: ${({ theme }) => theme.borderRadius.full};
+  box-shadow: 0 0 1.5rem 0.1rem rgba(102, 126, 234, 0.6);
+  color: ${({ theme }) => theme.color.white};
+  font-weight: 700;
+  letter-spacing: ${({ theme }) => theme.letterSpacing.wide};
+  margin-top: 2.5rem;
+  margin-bottom: 2.5rem;
```
Comment (lucaleivaloop) [inline src/components/ErrorBoundary/ErrorBoundary.styles.js:41]: `margin-bottom` should be over `margin-top`, you have the same error on `ErrorMessage`

Comment (mwolman) [reply src/components/ErrorBoundary/ErrorBoundary.styles.js:41]: When did we decide to order CSS properties alphabetically?

Comment (Tiago-Zarzavidjian) [reply src/components/ErrorBoundary/ErrorBoundary.styles.js:41]: I'll leave it like this:
`  margin: 2.5rem  0;`

**Pull request merged**